src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -399,6 +400,10 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt2.targets" />
 <Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.SkipCases.projitems" />
 
+<Target Name="_WriteLockFile">
+    <WriteLockFile LockFile="$(IntermediateOutputPath).__lock" />
```

Comment:
Should we be using `CondinueOnError` here? Just in case something horrible goes wrong... If this task errors when the entire build will stop.